packages/migrate/src/__tests__/DbPull.test.ts

```diff
@@ -169,9 +169,11 @@ describe('common/sqlite', () => {
     const introspect = new DbPull()
     const result = introspect.parse(['--print', '--url', 'postgresql://root:prisma@/prisma'])
     await expect(result).rejects.toThrowErrorMatchingInlineSnapshot(`
-            Error parsing connection string: empty host in database URL
+      P1013
 
-          `)
+      The provided database string is invalid. empty host in database URL. Please refer to the documentation in https://www.prisma.io/docs/reference/database-reference/connection-urls for constructing a correct connection string. In some cases, certain characters must be escaped. Please check the string for any illegal characters.
```

Comment:
This message is not optimal, but I guess unrelated to this PR and also not super important.

Reply:
Yep, this is unrelated to this PR